docs/OPERATOR_GETTING_STARTED.md

```diff
@@ -101,4 +80,49 @@ $ wget https://raw.githubusercontent.com/ethereum/eth2-ssv/stage/install.sh
 $ chmod +x install.sh
 
 $ ./install.sh
-```
\ No newline at end of file
+```
+
+### 4. Create Operator Keys
```

Comment:
creating operator key is not a must for the user. when ssv node is init - if there is no exist operator keys the node is auto generating the keys and write to storage. 
i think this step should be optional and explained why